compiler/src/dotty/tools/dotc/config/CliCommand.scala

```diff
@@ -3,6 +3,8 @@ package config
 
 import scala.language.unsafeNulls
 
+import dotty.tools.dotc.reporting.{ConsoleReporter}
```

Comment:
```suggestion
import dotty.tools.dotc.reporting.ConsoleReporter
```

We avoid the `{`/`}` when not needed.